metron-platform/metron-parsing/metron-parsers/src/main/java/org/apache/metron/parsers/cef/CEFParser.java

```diff
@@ -45,7 +45,7 @@
 	private static final Charset UTF_8 = Charset.forName("UTF-8");
 
 	private Pattern p;
-	private Pattern pext;
+	private static final Pattern pext = Pattern.compile(EXTENSION_CAPTURE_PATTERN);
```

Comment:
sorry, can we give this a better name too

Reply:
I kinda left that as out of scope since it's in the CEF parser, not the LEEF parser (the LEEF version doesn't use that technique). I'll fix it anyway, although I would say it was a bit of out of the scope of this work.

Reply:
you are right, my mistake.  But if we have already touched the file, might as well, but if you don't no biggie